### PR TITLE
Pr/cdr 2212 add switch component

### PR DESCRIPTION
**Pull request by KenjiCrosland**

### CODE REVIEW

**Comment by benjag** [comment]
I think your style assets need some work, anything that's currently a pixel should be a rem

File: src/components/switch/styles/vars/CdrSwitch.vars.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@mixin cdr-switch-base-mixin() {
+
+}
```
Comment (benjag) [inline src/components/switch/styles/vars/CdrSwitch.vars.scss:3]: Empty?

Comment (KenjiCrosland) [reply src/components/switch/styles/vars/CdrSwitch.vars.scss]: Removed

File: src/components/switch/CdrSwitch.vue [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  fullWidth: {
+    type: Boolean,
+    default: false
+  },
+  checked: {
+    type: Boolean,
+    default: false
+  }
+})
+//Safari focus state
```
Comment (benjag) [inline src/components/switch/CdrSwitch.vue]: Delete?

**Review by benjag — APPROVED** [review]

**Pull request merged**